packages/web-app-files/src/components/SideBar/Versions/FileVersions.vue

```diff
@@ -1,47 +1,50 @@
 <template>
-  <div id="oc-file-versions-sidebar">
+  <div id="oc-file-versions-sidebar" class="-oc-mt-s">
     <oc-loader v-if="loading" />
-    <oc-table-simple v-if="!loading && hasVersion">
-      <oc-tbody>
-        <oc-tr v-for="(item, index) in versions" :key="index" class="file-row">
-          <oc-td width="shrink" data-testid="file-versions-file-icon">
-            <oc-resource-icon :resource="resource" size="medium" />
-          </oc-td>
-          <oc-td
-            width="shrink"
-            class="oc-text-muted oc-text-nowrap"
-            data-testid="file-versions-file-last-modified-date"
-          >
-            {{ formatVersionDate(item) }}
-          </oc-td>
-          <oc-td class="oc-text-muted oc-text-nowrap" data-testid="file-versions-file-size">
-            {{ formatVersionFileSize(item) }}
-          </oc-td>
-          <oc-td v-if="isRevertable" width="shrink">
+    <ul v-if="!loading && hasVersion" class="oc-m-rm oc-position-relative">
+      <li class="spacer oc-pb-l"></li>
```

Comment:
Please add `aria-hidden="true"`. This element is only for styling, probably confusing for screen readers to have an empty list item.

Reply:
done